### PR TITLE
bidirectional sync post

### PR DESCRIPTION
**Pull request by mattmanning**
I don't like this title. Who has a better one?

Is the tone right here? I don't want to undersell this. I want to get the message across that Convox + Docker is better than Docker alone without sounding like I'm trashing them.

## Release Playbook
- [x] Rebase against master
- [x] Code review
- [x] Merge into master
- [ ] Verify changes at http://site-staging.convox.com
- [ ] Promote release on `site-production` in the `convox/production` Rack

### CODE REVIEW

**Comment by mattmanning** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;sTeo6SRd&#x2F;222-blog-post-for-2-way-sync"> Blog post for 2-way sync</a>

File: _posts/2016-06-07-2-way-sync.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+### Linux
+
+```
+$ curl -Ls https://install.convox.com/linux.zip > /tmp/convox.zip
+$ unzip /tmp/convox.zip -d /usr/local/bin
+```
+
+You’ll also need Docker installed locally. If you’re on OS X be sure to sign up for the Docker for Mac beta.
+
+Check out our Rails example from GitHub, `cd` into the directory and run `convox start`. Edit a file locally and observe the changes in the running container. `docker exec` onto `web` and generate a scaffold. Note that the new/changed files also appear on your local filesystem.
```
Comment (mwarkentin) [inline _posts/2016-06-07-2-way-sync.md]: Link to rails example?

Comment (mwarkentin) [inline _posts/2016-06-07-2-way-sync.md]: Or perhaps even better, change to a multi-line code example - something like:

```
git clone <rails repo>
cd rails
convox start
```

New tab:

```
docker exec rails-web rake ...
```

Comment (mattmanning) [inline _posts/2016-06-07-2-way-sync.md]: @mwarkentin Great suggestions, thanks.

File: _posts/2016-06-07-bidirectional-sync.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+```
+git clone git@github.com:convox-examples/rails.git
+cd rails
+convox start
+```
+
+Once the app is running you can run `docker ps` to get a list of container IDs and then exec onto a container and run Rails commands:
+
+```
+docker exec -it <container ID> bash
```
Comment (mwarkentin) [inline _posts/2016-06-07-bidirectional-sync.md]: I think the container will have a name: `rails-web` in this case

That'd be easier than using `docker ps` / `docker exec <container id>`.

**Pull request merged**